main.tf

```
@@ -0,0 +1,116 @@
+# Declare some local variables
+locals {
+  build_environment = var.environment == "production" ? "Prod" : "Dev"
+  common_tags = {
+    Product      = "SAEB Platform"
+    ProductOwner = "Shaun Laughland"
+    CreatedBy    = "Terraform"
+  }
+}
+
+# Refer to existing Azure resource group
+data "azurerm_resource_group" "saeb" {
+  name     = "SAEB-AnalyticsPlatform-Sndbx"
+}
+
+resource "random_string" "random" {
+  length = 4
+  upper = false
+  special = false
+  number = false
+}
+
+
+# Create storage account for main input and curated CSV files as well as for synapse filesystem
+resource "azurerm_storage_account" "saeb_storage" {
+  name                     = "stsaebdevca01${random_string.random.result}"
```

We might have to loop back on our naming conventions unless we can include a dynamic environment component to the name.
```suggestion
  name                     = "stapsndbxca01${random_string.random.result}"
```